.github/workflows/test.yml

```
@@ -352,7 +419,7 @@ jobs:
             --minimum-failure-severity=warning
             --recursive
             --config=${{ github.workspace }}/examples/.tflint.hcl
-          fail_on_error: true
+          fail_level: error
```

actionlint (1.7.4)

input "fail_level" is not defined in action "reviewdog/action-tflint@v1". available inputs are "fail_on_error", "filter_mode", "flags", "github_token", "level", "reporter", "tflint_config", "tflint_init", "tflint_rulesets", "tflint_target_dir", "tflint_version", "working_directory"

(action)